configuration/items.md

```diff
@@ -206,7 +206,46 @@ Number Livingroom_Temperature "Temperature [%.1f °C]"
 The state of an Item depends on the Item type, the Channel bound to the Item, and internal or external events.
 A analogy can be drawn between the state of an Item and the value of a variable in a computer program.
 
-<!-- TODO add a complete description about the actual state of an Item, e.g. Initialization, UNDEF, Binding etc. -->
+{: #item-state}
+#### Item State
+
+This section provides information about what a user can expect regarding the behavior of the state of an Item
+
+-  Items are created with a state of `NULL`.
+The Item will remain in this state until it has been acted upon
+-  Subsequent to the creation of an Item, operations in openHAB such as a user interacting with the Item using the `Basic UI`, or a Binding updating the state of an Item will change the state of the Item
+-  Once an Item is being used, its state will depend upon the Thing to which it is linked, or upon the specifics of the Binding that created the Item.
+Example - The state of a Switch may change from `NULL` to `ON` or `OFF`
+-  A Binding may set the state of an Item to `UNDEF` if it looses communications with a Thing (for example, a Z-wave doorbell with a dead battery).
+The Binding may also set the state to `UNDEF` if an error exists in the binding configuration, or under other conditions
+-  An Item may be assigned a particular state as part of the execution of a [Rule]({{base}}/configuration/rules-dsl.html).
+See, particularly, [Manipulating Item States]({{base}}/configuration/rules-dsl.html#manipulating-item-states)
```

Comment:
This list doesn't follow the style guide defined by https://github.com/openhab/openhab-docs/blob/gh-pages/package.json

Reply:
Okay - I do not have your linter up and running yet.  Let me see if I can get it going and see what it reveals.

Reply:
Okay - ran linter and made a new commit fixing formatting errors.  (FWIW, also ran linter against the 'nit' pass and added a new commit to fix that as well.)